daemon/daemon.go

```diff
@@ -62,8 +62,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-// MainNamespace is the name of the namespace used for users containers
-const MainNamespace = "moby"
+// ContainersNamespace is the name of the namespace used for users containers
+const ContainersNamespace = "containers.moby"
```

Comment:
We can shorten this to just `moby`, and that will limit the impact.

Reply:
Arf, I like `containers.moby` here though 😓.
If we need to do a migration anyway, let's do this one right (and thus I would like to have `containers.moby` instead).